flink-libraries/flink-table/src/test/scala/resources/testUnionStream0.out

```diff
@@ -2,3 +2,21 @@
 LogicalUnion(all=[true])
   LogicalTableScan(table=[[_DataStreamTable_0]])
   LogicalTableScan(table=[[_DataStreamTable_1]])
+
+== Physical Execution Plan ==
+Stage 1 : Data Source
+	content : collect elements with CollectionInputFormat
+
+Stage 2 : Data Source
+	content : collect elements with CollectionInputFormat
+
+	Stage 3 : Operator
+		content : from: (count, word)
+		ship_strategy : REBALANCE
+		exchange_mode : null
+
+		Stage 4 : Operator
+			content : from: (count, word)
+			ship_strategy : REBALANCE
+			exchange_mode : null
```

Comment:
It seems that the json execution plan generated from `env.getExecutionPlan` dosen't contain the `exchange_mode` information. Could we remove this node in explain?